packages/cli/src/deploy/debug.ts

```diff
@@ -1,10 +0,0 @@
-import { debug as parentDebug } from "../debug";
-
-export const debug = parentDebug.extend("deploy");
-export const error = parentDebug.extend("deploy");
```

Comment:
I moved it to `packages/world/ts/debug.ts`, it is needed there as well in the helper functions that got moved. Should I keep two copies then, one in `cli`, one in `world`?

Reply:
yes, the loggers should always be local to their own source, not imported from another project, otherwise things using `debug` in CLI get logged out as if it was from the world package

Reply:
Got it, didn't think of that. Moved back